tests/cases/conformance/jsdoc/jsdocTemplateTagNameResolution.ts

```diff
@@ -0,0 +1,15 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+// @Filename: file.js
```

Comment:
Could you also enable declaration emit for this (or both) test(s)? JS declaration emit is pretty specialized, after all.